README.md

```diff
@@ -1,2 +1,159 @@
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# To-do List <a name="about-project"></a>
+
+**Project To Do List!** 
+Welcome to the  repository for our Project To do List!
+This repository contains the code and assets for To do list application."To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/Javascript">CSS</a></li>
+</ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **function to iterate over the tasks array and populate an HTML list item element for each task.**
+- **Engaging and interactive design**
+- **On page load, rendered the dynamically created list of tasks in the dedicated placeholder.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have NodeJS installed
+- A Web Browser (Google Chrome, Firefox, etc)
+- A Code Editor (Notepad++, VSCode, etc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone yourlink repo [https://github.com/the-faizmohammad/conferencePage]
+
+cd your folder name
+```
+
+### Install
+
+Install this project with:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+npm run build
+### Run tests
+
+To run tests, run the following command:
+
+npm run dev
+
+### Deployment
+
+Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Faiz Mohammad **
+
+- GitHub: [@githubhandle](https://github.com/the-faizmohammad)
+- Twitter: [@twitterhandle](https://twitter.com/FaizMoh58742600)
+- LinkedIn: [LinkedIn](https://twitter.com/FaizMoh58742600)
+- 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add a hello animated illustration**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
- [ Optional ] I think it would be good if you make sure that the `issues` link on your readme file leads to the `issues` page. Currently, the link redirects to the repository instead of the issues page. I would like to suggest that you refactor this link as shown below for it to work correctly.

## Issues link
Feel free to check the  [Issues](https://github.com/the-faizmohammad/Project-To-Do-List/issues)

```sh
   [Issues](https://github.com/the-faizmohammad/Project-To-Do-List/issues)
```